src/api/docker.ts

```diff
@@ -104,7 +103,11 @@ export function attachExec(id: string, cmd: any): Observable<any> {
         } else if (str.includes('[success]')) {
           exitCode = 0;
           ws.end();
-        } else if (!str.includes('/usr/bin/abstruse') && !str.startsWith('>')) {
+        } else if (!str.includes('/usr/bin/abstruse \'' + cmd.command) && !str.startsWith('>')) {
+          if (str.includes('//') && str.includes('@')) {
```

Comment:
isn't this already on inside `process.ts`?

Reply:
No. In `process.ts` there is just `observer.next(event);`.

Reply:
its in config.ts, I knew I saw this somewhere :)

Reply:
In `config.ts`, an access token is added to cloneUrl, here its the opposite, here is token removed from a string, so it's not shown in console output.